### DISCUSSION TITLE
Any way to see the list of channels a client is subscribed to or the list of clients subscribed to a channel?

### DISCUSSION THREAD

**Discussion by synzhu** [uNetworking/uWebSockets.js#305] [Q&A] [answered]
What is the easiest way for me to to see the list of channels a client is subscribed, as well as the list of clients subscribed to a channel?

For example, before allowing a client to publish to a channel, I may want to check that it is first subscribed to the channel

**Comment by mikeandtherest** [comment]
I think this was answered before. The only way, for now at least, to achieve that, is to track it yourself with lists/dictionaries.

**Comment by synzhu** [comment]
I see. The library itself must keep track of this information somewhere though, right? Is it just not possible to access this information?

**Comment by ghost** [comment, marked as answer]
It's not possible right now, but something like that could be added.
